miner/bid_simulator.go

```diff
@@ -64,6 +68,7 @@ var (
 type WorkPreparer interface {
 	prepareWork(params *generateParams) (*environment, error)
 	etherbase() common.Address
+	fillTransactions(interruptCh chan int32, env *environment, stopTimer *time.Timer, bidTxs *hashset.Set) (err error)
```

Comment:
I think it is not proper to call it `WorkPreparer` any longer as it has the `fillTransactions` interface.

Reply:
bidWorker now